internal/cluster/validations/validations.go

```diff
@@ -905,3 +911,23 @@ func ValidateHighAvailabilityModeWithPlatform(highAvailabilityMode *string, plat
 
 	return nil
 }
+
+func ValidateIgnitionImageSize(config string) error {
+	var err error
+	var data *bytes.Reader
+
+	// Ensure that the ignition archive isn't larger than 256KiB
+	configBytes := []byte(config)
+	content := isoeditor.IgnitionContent{Config: configBytes}
+	data, err = content.Archive()
+	if err != nil {
+		return errors.Wrap(err, fmt.Sprintf("Failed to create ignition archive (config: %s)", config))
+	}
+	ignitionImageSize := data.Len()
+	if ignitionImageSize > IgnitionImageSizePadding {
+		return errors.New(fmt.Sprintf("The ignition archive is too large: %d KiB. Its size should be up to %d KiB",
```

Comment:
Maybe `The ignition archive size (%d KiB) is over the maximum allowable size (%d KiB)`?

Reply:
Sure, sounds good.